doc/development_guide/how_tos/custom_checkers.rst

```diff
@@ -218,11 +218,26 @@ Now we can debug our checker!
 .. Note::
 
     ``my_plugin`` refers to a module called ``my_plugin.py``.
-    This module can be made available to pylint by putting this
-    module's parent directory in your ``PYTHONPATH``
+    The preferred way of making this plugin available to pylint is
+    by installing it as a package, such as with ``pip install .``.
```

Comment:
```suggestion
    by installing it as a package. This can be done either from a packaging index like
    ``PyPI`` or by installing it from a local source such as with ``pip install .``.
```